### PR TITLE
browser: contentcontrol: add date type

### PR DESCRIPTION
**Pull request by Rash419**
add uno command to insert date in form tab
renamed 'Content Control Properties' -> 'Properties'

Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: Ib4a439cd349bb79430d590e0b80c825372c629d3

* Target version: master

### CODE REVIEW

**Comment by Rash419** [comment]
@pedropintosilva @Andreas-Kainz we need two more icons for picture and date content control : )

**Review by vmiklos — APPROVED** [review]
I tested this, it works for me: I can insert a date content control, it gets rendered, I can select a date and it gets set on the content control. Ideally please wait for a review from @gokaysatir (or somebody else) on the JS style bits. Thanks!

**Review by pedropintosilva — APPROVED** [review]

**Pull request merged**
